xarray/core/datatree.py

```diff
@@ -567,52 +675,7 @@ def sizes(self) -> Mapping[Hashable, int]:
         --------
         DataArray.sizes
         """
-        return self.dims
-
-    @property
-    def _attr_sources(self) -> Iterable[Mapping[Hashable, Any]]:
-        """Places to look-up items for attribute-style access"""
-        yield from self._item_sources
-        yield self.attrs
-
-    @property
-    def _item_sources(self) -> Iterable[Mapping[Any, Any]]:
-        """Places to look-up items for key-completion"""
-        yield self.data_vars
```

Comment:
I got horrible recursion errors from the attribute-like access method, and I just took the shortcut of deleting it for now.